glide-core/src/client/value_conversion.rs

```diff
@@ -319,11 +340,15 @@ fn convert_array_elements(
     Ok(Value::Array(converted_array))
 }
 
-fn convert_array_to_map(
+fn convert_array_to_map<F, G>(
     array: Vec<Value>,
-    key_expected_return_type: Option<ExpectedReturnType>,
-    value_expected_return_type: Option<ExpectedReturnType>,
-) -> RedisResult<Value> {
+    array_push_fn: F,
+    other_push_fn: G,
```

Comment:
aren't F and G defined the same?  Couldn't we do: 
```
    array_push_fn: F,
    other_push_fn: F,
```

Reply:
good point. Alternatively, you can use a trait so we don't have to pass 2 closures. Up to you though.

Reply:
This is fine.  Thanks.